kubtest-dashboard/charts/kubtest-dashboard/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
```

Comment:
Hey @nicufk ,
What is the purpose of this POD?
What will happen if it fails for example?

Reply:
It was generated by helm create.